src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,60 @@
+import { FilterStatus } from '../../types/Filterstatus';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  completedTodos: Todo[] | [],
+  onStatusChange: (str: FilterStatus) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  completedTodos,
+  onStatusChange,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        4 items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className="filter__link selected"
```

Comment:
You should hightlight this link only when you filtering your data by this parametr. The same situation for another links.